controller/argument_value_resolver.rst

```diff
@@ -233,8 +233,8 @@ and adding a priority.
 
         use App\ArgumentResolver\UserValueResolver;
 
-        return static function (ContainerConfigurator $container) {
-            $services = $configurator->services();
```

Comment:
Found some places where variables does not match, `$container` injected but `$configurator` used.